lib/cruddy-food.js

```diff
@@ -111,38 +112,27 @@ class CruddyFood {
   }
 
   deleteMealFoodListener() {
-    $('.mealsDeleteButton').on('click', function() {
+    let meals = ['Breakfast', 'Lunch', 'Dinner', 'Snack']
+      meals.forEach(function(meal) {
+    $(`.${meal}DeleteButton`).on('click', function() {
```

Comment:
Nice, forgot to update this previously. The html tags are, for instance 'BreakfastDeleteButton` so this was previously never catching